Tasks/NuGetInstaller/NuGetInstaller.ps1

```diff
@@ -8,6 +9,11 @@ param(
     [string]$nuGetPath
 )
 
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
+
+. $PSScriptRoot\VsoNuGetHelper.ps1
```

Comment:
Given the verbose tracing immediately below (i.e. "Entering script [...] Parameter values [...]"), I think it would make more sense and for troubleshooting sake to move your new line down instead of the import module statements up.